### DISCUSSION TITLE
Manually specify notable contributions

### DISCUSSION THREAD

**Discussion by queensferryme** [lowlighter/metrics#550] [Ideas]
It would be nice if the _notable_ plugin allows manually specifying a list of repos like:
```yaml
plugin_notable_custom: owner1/repo1, owner2/repo2
```

Some motives behind this proposal:
1. Currently only orgnizations are included in notable contributions. However, there are a lot of awesome projects hosted by a personal account (including this one!)
2. Some contributions are not that notable. Users may want to exclude contributions like `fix typo` etc.

**Comment by lowlighter** [comment]
Hello, thanks for your suggestions 🙂 !

1. I'll check to add support for repositories hosted by personal accounts. Probably will iterate over the GitHub search syntax filtering (`plugin_notable_filter`) to add support for users and repositories. I prefer improving the automation rather than providing manual options where it is possible (which seems to be in this case). Do you think it would still fulfill your request ? 
2. It may be hard to implement. I'm actually not sure that GraphQL edges includes the commits of the user, so it's not possible to list them directly and load their messages to filter them like you suggested (I'm sure there a way to do it, but probably needs some work or consumes a lot of API requests). 

This plugin was mostly designed to highlight contributions to large open source repositories (usually hosted by organizations, though like you mentionned some are hosted by users) so even small ones have some merit (even typos). I agree some are more important than others, but ultimately if they're listed by this plugin it means that the maintainers judged a contribution useful 🙂 

Edit: 1. has been implemented in #560
